packages/devextreme/scss/widgets/base/_htmlEditor.scss

```diff
@@ -180,11 +180,14 @@ $transparent-border: 1px solid transparent;
   h4,
   h5,
   h6,
-  ol,
   ul {
     counter-reset: add-counter-reset($max-indent);
   }
 
+  ol {
+    counter-reset: list-item 0;
+  }
+
```

Comment:
Removing ol on line 183 fixed Firefox bug but this caused problem for the other browsers. That's why I explicitly assigned counter-reset: list-item 0.